Modules/ITS/itsQCTrendingTracks.json

```diff
@@ -44,6 +71,62 @@
             "varexp": ["NClusters.mean:ntreeentries", "NClusters.stddev:ntreeentries"],
             "selection": "",
             "option": "PL"
+          },
+          {
+            "names": ["EtaDistribution_mean", "EtaDistribution_stddev"],
+            "title": ["Average EtaDistribution trend", "Stddev EtaDistribution trend"],
+            "varexp": ["EtaDistribution.mean:ntreeentries", "EtaDistribution.stddev:ntreeentries"],
+            "selection": "",
+            "option": "PL"
+          },
+          {
+            "names": ["PhiDistribution_mean", "PhiDistribution_stddev"],
+            "title": ["Average PhiDistribution trend", "Stddev PhiDistribution trend"],
+            "varexp": ["PhiDistribution.mean:ntreeentries", "PhiDistribution.stddev:ntreeentries"],
+            "selection": "",
+            "option": "PL"
+          },
+          {
+            "names": ["VertexZ_mean", "VertexZ_stddev"],
+            "title": ["Average VertexZ trend", "Stddev VertexZ trend"],
+            "varexp": ["VertexZ.mean:ntreeentries", "VertexZ.stddev:ntreeentries"],
+            "selection": "",
+            "option": "PL"
+          },
+          {
+            "names": ["NVertexContributors_mean", "NVertexContributors_stddev"],
+            "title": ["Average NVertexContributors trend", "Stddev NVertexContributors trend"],
+            "varexp": ["NVertexContributors.mean:ntreeentries", "NVertexContributors.stddev:ntreeentries"],
+            "selection": "",
+            "option": "PL"
+          },
+          {
+            "names": ["AssociatedClusterFraction_mean", "AssociatedClusterFraction_stddev"],
+            "title": ["Average AssociatedClusterFraction trend", "Stddev AssociatedClusterFraction trend"],
+            "varexp": ["AssociatedClusterFraction.mean:ntreeentries", "AssociatedClusterFraction.stddev:ntreeentries"],
+            "selection": "",
+            "option": "PL"
+          },
+          {
+            "names": ["Ntracks_mean", "Ntracks_stddev"],
+            "title": ["Average Ntracks trend", "Stddev Ntracks trend"],
```

Comment:
In the title put: `Average NTracks per event trend` and `Stddev of NTracks per event`